### PR TITLE
Mock Smartcard Tweaks

### PR DESCRIPTION
**Pull request by adghayes**
Makes two tweaks to the `mockCardReader.py` script for easier development, as separate commits.

First, allows the `mockCardReader.py` to accept a `--dip` flag which will result in the card getting removed shortly after being inserted. Only makes sense to use alongside the `SKIP_PIN_ENTRY` flag. 

Second, adds the ability to mock long values on poll worker cards. This is important in developing report printing flows on VxMark. I've added fixtures for one election definition only.

### CODE REVIEW

File: services/smartcards/writePollWorkerCard.py
Comment (eventualbuddha) [inline services/smartcards/writePollWorkerCard.py:17]: Can we either add a comment explaining what the expected args are and what they do, or handle `--help`/`-h` args to print expected usage?

File: services/smartcards/mockCardReader.py
Comment (eventualbuddha) [inline services/smartcards/mockCardReader.py:241]: Add `--dip` to the usage info above, along with an explanation and example.

**Review by eventualbuddha — APPROVED** [review]
Needs a couple tweaks, but LGTM after.

File: services/smartcards/writePollWorkerCard.py
Comment (adghayes) [reply services/smartcards/writePollWorkerCard.py:18]: Comment!

**Pull request merged**
